### PR TITLE
Add in blog post on changes to OSTree ref updates.

### PR DESCRIPTION
**Pull request by dustymabe**

### CODE REVIEW

**Comment by cgwalters** [comment]
I would state this from a high level:

"The default Fedora cadence for updates in the RPM streams is once a day.  Until now, the OSTree-based updates cadence has matched this.  However, we're changing the default OSTree update stream to match the image release cadence (once every two weeks).

If you want to access the once-a-day updates, you can use <new ref>".

File: source/blog/2017-02-10-matching-fedora-ostree-released-content-with-each-2week-atomic-release.html.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+what if you already have an Atomic Host server up and running? 
+
+READMORE
+
+Servers that are already running are configured to get their updates
+directly from the OSTree repo that is sitting on Fedora Infrastructure
+servers. The client will ask "What is the newest commit for my
+branch/reference?" and the server will kindly reply with the most recent commit.
+If the client is at an older version then it will start to pull the
+newer commit and will apply the update.
```
Comment (cgwalters) [inline source/blog/2017-02-10-matching-fedora-ostree-released-content-with-each-2week-atomic-release.html.md]: Too detailed/obtuse IMO.

File: source/blog/2017-02-10-matching-fedora-ostree-released-content-with-each-2week-atomic-release.html.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+This is exactly how the client is supposed to behave, but one problem
+with the way we have been doing things in the past is that we have
+been updating everyones "branch/reference" every night when we do
+our updates runs in Fedora.
+
+This has the side effect of meaning that users can get content as soon
+as it has been created, but it also means that the two week release
+process where we perform testing and validation really means nothing
+for these users, as they will get something before we ever did testing
+on it.
```
Comment (cgwalters) [inline source/blog/2017-02-10-matching-fedora-ostree-released-content-with-each-2week-atomic-release.html.md]: I'd replace all three of these paragraphs with something like my suggested paragraph above.

File: source/blog/2017-02-10-matching-fedora-ostree-released-content-with-each-2week-atomic-release.html.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+process where we perform testing and validation really means nothing
+for these users, as they will get something before we ever did testing
+on it. 
+
+We have decided to slow down the cadence of the
+"fedora-atomic/25/x86_64/docker-host" ref within the OSTree
+repo to match the exact releases that we do for the two week release
+process. Users will be able to track this ref like they always have,
+but it will only update when we do a release, approximately every
+two weeks.
```
Comment (cgwalters) [inline source/blog/2017-02-10-matching-fedora-ostree-released-content-with-each-2week-atomic-release.html.md]: This is the important one.

File: source/blog/2017-02-10-matching-fedora-ostree-released-content-with-each-2week-atomic-release.html.md (hunk 2 above)
Comment (dustymabe) [inline source/blog/2017-02-10-matching-fedora-ostree-released-content-with-each-2week-atomic-release.html.md]: the only problem with this approach is it doesn't capture any of the "why". Why don't I make your paragraph a TL;DR at the beginning and then let the rest of the text explain the "why" ?

**Comment by dustymabe** [comment]
@cgwalters - added your paragraph as TL;DR at the top of the article, can you review?

File: source/blog/2017-02-10-matching-fedora-ostree-released-content-with-each-2week-atomic-release.html.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+our updates runs in Fedora.
+
+This has the side effect of meaning that users can get content as soon
+as it has been created, but it also means that the two week release
+process where we perform testing and validation really means nothing
+for these users, as they will get something before we ever did testing
+on it. 
+
+We have decided to slow down the cadence of the
+`fedora-atomic/25/x86_64/docker-host` ref within the OSTree
```
Comment (miabbott) [inline source/blog/2017-02-10-matching-fedora-ostree-released-content-with-each-2week-atomic-release.html.md]: At the risk of being pendantic, you use 'ref' for the first time here but in the question in line 26, you use 'branch/reference'.  I'd pick one and align all of the text with it.

**Comment by miabbott** [comment]
One minor comment from me inline, but otherwise looks good.

**Comment by dustymabe** [comment]
> One minor comment from me inline, but otherwise looks good.

moved all 'reference' to 'ref'.

**Comment by jberkus** [comment]
Please add this example to the bottom of the article:

https://gist.github.com/jberkus/75168b0c0cd3656c4fd8f57d1a3028ba

**Comment by dustymabe** [comment]
@jberkus - i added the content but fixed it a bit so that we actually showed the ref from the last two week release `f294635a1dc62d9ae52151a5fa897085cac8eaa601c52e9a4bc376e9ecee11dd`.

**Pull request merged**